be/src/vec/functions/array/function_array_enumerate_uniq.cpp

```diff
@@ -95,15 +95,13 @@ class FunctionArrayEnumerateUniq : public IFunction {
                         "The {} -th argument for function: {} .must be an array but it type is {}",
                         i, get_name(), arguments[i]->get_name());
             }
-            if (i == 0) {
-                is_nested_nullable = array_type->get_nested_type()->is_nullable();
-            }
+            is_nested_nullable = is_nested_nullable | array_type->get_nested_type()->is_nullable();
```

Comment:
Do you mean `||` instead of `|` ?

Reply:
``` || ``` and ``` | ``` all make the res right, I intended to use bit operation, but ``` || ``` can short-circuit phrase, I will change it